Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -301,6 +301,7 @@ static _WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
 
     unloadBackgroundWebView();
     removeInjectedContent();
+    m_registeredScriptsMap.clear();
```

Comment:
Should this move to the `removeInjectedContent()` method?

Also there are other places were we might want to remove these scripts, the other `removeInjectedContent` methods.

Reply:
Actually this is correct, the `removeInjectedContent()` is called in `updateInjectedContent() { removeInjectedContent(); addInjectedContent(); }` and this would be incorrect to do there.